The commit
https://github.com/openlink/virtuoso-opensource/commit/9cececaca5df32c82576e5390062475bbf5e1cc1
added a definition of GIT_HEAD_STR in one place but didn't handle
the case where `git_head.c` didn't exist and the build was not run out of
a Git repository.

diff --git a/libsrc/Wi/mkgit_head.sh b/libsrc/Wi/mkgit_head.sh
index f34107b..4fa0261 100755
--- a/libsrc/Wi/mkgit_head.sh
+++ b/libsrc/Wi/mkgit_head.sh
@@ -4,7 +4,8 @@ VALUE=000000
 
 if test \! -f git_head.c 
 then
-    echo "char * git_head = \"$VALUE\";" > git_head.c
+    echo "#define GIT_HEAD_STR \"$VALUE\"" > git_head.c
+    echo "char * git_head = \"$VALUE\";" >> git_head.c
 fi
 
 VALUE=`git rev-parse --verify HEAD --short` || exit
